src/content/topics/home/experimentation-creating/index.mdx

```diff
@@ -134,12 +134,20 @@ To start an experiment iteration:
 3. Click **Start**.
 4. Repeat steps 1-3 for each environment you want to start an iteration in.
 
-![The "Results" tab of an experiment with the "Start" button called out.](experiment-results-start-callout.png)
+![An experiment with the "Start" button called out.](experiment-details-start-callout.png)
 
 Experiment iterations allow you to record experiments in discrete blocks of time. To ensure accurate experiment results, when you make changes that impact an experiment, LaunchDarkly starts a new iteration of the experiment.
 
 To learn more about starting and stopping experiment iterations, read [Managing experiments](/home/analyzing-experiments/managing).
 
+## Archiving Experiments
+
+You can archive an experiment if you want to retire it from LaunchDarkly.
+
+Archived experiments are hidden from the experiment dashboard by default, however you still cannot delete a flag or metric attached to an archived experiment.
+
+![An experiment with the "Archive experiment" button called out.](experiment-archive-callout.png)
+
```

Comment:
```suggestion
## Archiving and restoring experiments

You can archive an experiment if you want to retire it from LaunchDarkly. 

Archived experiments are hidden from the experiment dashboard by default. However, you cannot delete a flag or metric attached to an archived experiment. You also cannot start new iterations for archived experiments.

To archive an experiment:

1. Navigate to the **Experiments** list in the environment you want to start an iteration in.
2. Click on the name of the experiment you want to archive. The experiment detail page appears.
3. Click **Archive experiment**.

![An experiment with the "Archive experiment" button called out.](experiment-archive-callout.png)

To restore an experiment, click **Restore experiment** from the experiment details page. 

```

Suggested updates to:
- use sentence case in section title
- include short procedure on archiving steps
- note that you cannot start iterations for archived experiments
- note that you can restore experiments

For restoring experiments, I noticed this in the tech spec and played with it in staging. If it won't be released at the same time, let's remove that line for now.

Reply:
oh nice, thanks @mmrj! Restoring _will_ be released at the same time, so we can leave that in :)